app/views/covid19_mailer/covid_19_delay.text.erb

```diff
@@ -0,0 +1,7 @@
+Dear <%= @application_form.first_name || 'candidate' %>
+
+Given the impact of coronavirus (COVID-19) on schools and universities, your teacher training application might be processed more slowly than usual.
+
+To check the status of your application, sign in to Apply for teacher training https://www.apply-for-teacher-training.education.gov.uk/candidate/sign-in
+
+If you have any questions about this, you can contact us at [becomingateacher@digital.education.gov.uk](mailto:becomingateacher@digital.education.gov.uk)
```

Comment:
Had to type this out as I couldn't see anything other than a screenshot of the copy on the Trello card. Would appreciate a proof-read.

Reply:
Looks good!

Reply:
Did a proofread against the screenshot. Looks right to my eyes! 🕵️‍♀️